src/icons/_icon-common-style.scss

```diff
@@ -11,3 +11,7 @@
 .icon {
     @include icon-classes-generator(); // (mixin)
 }
+
+.icon-white {
+    filter: brightness(1000%);
```

Comment:
Hmm... this is probably intended, but the code looks like at bug; is there another way to do this?

Reply:
There's always other ways, but only more cumbersome ones...

Reply:
Ok, maybe drop in [sic]-comment then ;)